clean_user_registry_secrets/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.7-alpine
+
+LABEL maintainer="info@datascience.ch"
+
+RUN apk update && \
```

Comment:
I think `apk update` is not needed when you use the `--no-cache` option for the install command.